builtin/logical/pki/path_root.go

```diff
@@ -257,13 +257,16 @@ func (b *backend) pathCAGenerateRoot(ctx context.Context, req *logical.Request,
 
 	// Also store it as just the certificate identified by serial number, so it
 	// can be revoked
+	b.certsCountedLock.RLock() // Don't allow storage of certs during initial count
 	err = req.Storage.Put(ctx, &logical.StorageEntry{
 		Key:   "certs/" + normalizeSerial(cb.SerialNumber),
 		Value: parsedBundle.CertificateBytes,
 	})
+	b.certsCountedLock.RUnlock() // Unlock before we build the CRL
```

Comment:
My thinking here (on not using a defer) is that CRL Building can be pretty long, but if others feel like a defer is still better, I'd change it back.

Reply:
I think CRL building can be _reallly_ long so I like not holding this lock (if we keep this lock).

Reply:
Thank you - but this lock is removed.  :(